Reason: Skip npm global install test which fails on arches besides x86_64
and aarch64, limited by npm module lightningcss
Upstream: No

diff --git a/common/hexec/exec_integration_test.go b/common/hexec/exec_integration_test.go
index 3c820b570..6c3ef2c13 100644
--- a/common/hexec/exec_integration_test.go
+++ b/common/hexec/exec_integration_test.go
@@ -14,15 +14,15 @@
 package hexec_test
 
 import (
+	"os"
 	"testing"
 
-	"github.com/gohugoio/hugo/htesting"
 	"github.com/gohugoio/hugo/hugolib"
 )
 
 func TestNPMGlobalInstalls(t *testing.T) {
-	if !htesting.IsRealCI() {
-		t.Skip("We only ever want to run this in CI.")
+	if os.Getenv("GOARCH") != "amd64" && os.Getenv("GOARCH") != "arm64" {
+		t.Skip("Skip npm global install test.")
 	}
 	files := `
 -- hugo.toml --
